docs/installation.rst

```diff
@@ -317,12 +317,12 @@ Use the Cygwin installer to install the dependencies::
 
     python3 python3-devel libcrypt-devel python3-setuptools
```

Comment:
removed libcrypt-devel (borg does not need that directly, but likely python itself will pull that as a dependency).

i used the pkg38-xxx names because they all follow the same pattern and some stuff has no metapackage python3-xxx.